chains/v10/chains.json

```diff
@@ -6048,5 +6048,42 @@
         "options": [
             "ethereumBased"
         ]
+    },
+    {
+        "chainId": "f4f7be86702d82728d97c3f969d93697b3a7c415afca098e00858f68f96dc28f",
+        "name": "TSCS",
+        "assets": [
+            {
+                "assetId": 0,
+                "symbol": "TSCS",
+                "precision": 18,
+                "priceId": "tscs-network-native-token",
+                "icon": "https://raw.githubusercontent.com/novasamatech/nova-utils/master/icons/chains/white/TSCS.svg"
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://testnetrpc.scschain.com",
```

Comment:
thx for your PR, to apply testnet you can use add network feature manually: https://docs.novawallet.io/nova-wallet-wiki/settings/network-management/add-network-manually